src/analysis/out_parameters.rs

```diff
@@ -96,9 +96,9 @@ pub fn analyze_imports(env: &Env, func: &Function, imports: &mut Imports) {
     for par in &func.parameters {
         if par.direction == ParameterDirection::Out {
             match *env.library.type_(par.typ) {
-                Type::Fundamental(..) |
-                    Type::Bitfield(..) |
+                Type::Bitfield(..) |
                     Type::Enumeration(..) => imports.add("std::mem", func.version),
+                Type::Fundamental(fund) if fund != Fundamental::Utf8 && fund != Fundamental::Filename => imports.add("std::mem", func.version),
```

Comment:
Where would you like it to be split? :) Before the `if` or before the `=>`?

It might make sense to run everything through the new rustfmt (rustfmt-nightly) at some point to make everything consistent.